.travis.yml

```diff
@@ -0,0 +1,13 @@
+language: c
+
+services:
+        - docker
+
+before_install:
+        - docker pull ghdl/ext:latest
+
+os:
```

Comment:
Since this is the default, you might omit it. Furthermore, the docker container should work on any host platform.